.github/workflows/ci.yml

```diff
@@ -62,4 +62,4 @@ jobs:
           yarn install --immutable --immutable-cache
       - name: Check Formatting
         run: |
-          yarn run nx format:check
+          yarn run nx format:check --base=master
```

Comment:
the action doesnt checkout `master`, so this is the same as before
CI minutes are free here, so we might as well keep checking all files with `yarn run nx format:check --all` to be cautious against extra CRLFs or other issues creeping